### ISSUE TITLE
~/.addic7edrc doesn't work

### ISSUE THREAD

**Issue by gregaou** [Jesus-21/addic7ed#1]
```
$ cat ~/.addic7edrc 
[config]
lang = 8
```

I've tried to set my language in a .addic7edrc file but that make no effects when i run addic7ed

**Comment by gregaou** [comment]
2 solutions:

Your example should be 

```
$ cat ~/.addic7edrc 
[addic7ed]
lang = 8
```

or in https://github.com/Jesus-21/addic7ed/blob/master/addic7ed/constants.py#L64

```
if "config" in config and "lang" in config["config"]:
    LANG = config["config"]["lang"]
```

I will use temporaly? the first solution

**Comment by pogman-code** [comment]
Closed by e94cc77f252096780eeb9537ba89accf0902abb6
